XenAdmin/Controls/XenSearch/QueryPanel.cs

```diff
@@ -213,12 +213,12 @@ private static GridItemBase GetDefaultColumn(string s)
 
             if (s == "disks")
             {
-                return NewGridHeaderItem(Messages.OVERVIEW_DISKS, SortOrder.Descending, 120, 120);
+                return NewGridHeaderItem(Messages.OVERVIEW_DISKS, SortOrder.Descending, 138, 138);

```

Comment:
This number is hardcoded, which means it's the same for all languages and I'm a bit concerned that it may result to the english counterpart having more white space than needed. Ideally, this number should be somewhere in a localisable resource file (perhaps on the NewGridHeaderItem or parent classes).

Can we check how easy it is to do this - these are completely custom controls and there may be caveats, for example, one thing to watch out for is how that resource manager will be called to avoid memory leaks like we had recently with the images on the command toolstrip items. If it's too big a change, ok, let's leave it.